File: submissions/sm_014_kondreddy/week_02/day_03/session_02/average_divisible.js
var sum , avg , count,num ;
count = 0;
sum = 0;
avg = 0;
num= 4;
for(i = 1;i < 20;i++) {
    if(i%num == 0) {
    sum = sum+i;
    count = count +1;
    // console.log(i)
    }
}
avg = sum/count;
    console.log(avg)

File: submissions/sm_010_dipanshu/week_13/day_4/session_1/average_divisible.py
limit = 30
num = 3

sum_of_factors = 0
count = 0
avg = 0

for i in range(1, limit + 1):
    if i % num == 0:
        sum_of_factors += i
        count += 1

avg = sum_of_factors / count
print(avg)
